src/map/addLayer.js

```diff
@@ -52,7 +52,7 @@ export default function (layer, mapConfig) {
     layerObj.legendBottom = 40;
   }
 
-  if (layer.property) {
+  if (layer.property && layer.categories) {
```

Comment:
Interesting, has `layer.categories` been a requirement for stops this whole time??